init.lua

```diff
@@ -970,8 +988,10 @@ PaperWM.actions = {
     swap_down = partial(PaperWM.swapWindows, PaperWM, Direction.DOWN),
     center_window = partial(PaperWM.centerWindow, PaperWM),
     full_width = partial(PaperWM.setWindowFullWidth, PaperWM),
-    cycle_width = partial(PaperWM.cycleWindowSize, PaperWM, Direction.WIDTH),
-    cycle_height = partial(PaperWM.cycleWindowSize, PaperWM, Direction.HEIGHT),
+    cycle_width = partial(PaperWM.cycleWindowSize, PaperWM, Direction.WIDTH, "ascending"),
```

Comment:
I prefer to use a constant enum for commands, instead of matching strings. Could you add an `ASCENDING` and `DESCENDING` member to the `Direction` enum? Please also check if `cycle_direction` is `Direction.ASCENDING` or `Direction.DESCENDING` before changing the window size and error out of that method if not.